DAQ/test/BinaryPacketsFromDataBlocksTest.fcl

```diff
@@ -5,62 +5,70 @@
 #
 # Contact person Tomonari Miyashita
 
+#include "fcl/minimalMessageService.fcl"
+#include "fcl/standardServices.fcl"
+
 process_name : FileDumper
 
+
+services : @local::Services.Reco
+
 source : {
   module_type : RootInput
-  fileNames   : @nil
+    fileNames   : @nil
 
-  # firstEvent  : 500
-  maxEvents   : -1
+    # firstEvent  : 500
+    maxEvents   : -1
 }
 
 physics : {
 
-  producers : {
-    binaryOutput: {
-      module_type     : BinaryPacketsFromDataBlocks
-      diagLevel       : 2
-      maxFullPrint    : 0
-
-      includeTracker : 1
-      includeCalorimeter : 1
-      includeCosmicRayVeto: 1
-      includeDMAHeaders: 1
-      # Cosmic ray veto module is currently incomplete
-
-      generateTimestampTable : 1
-
-      generateBinaryFile : 1
-      
-      outputFile      : "DTC_packets.bin"
-      maxDMABlockSize : 32000
-      bufferSize      : 1000
+    producers : {
+	binaryOutput: {
+	    #      module_type     : BinaryPacketsFromDataBlocks
+	    module_type     : ArtBinaryPacketsFromDigis
+	    # diagLevel       : 2
+	    diagLevel       : 4
+	    maxFullPrint    : 0
+
+	    includeTracker : 0
+	    includeCalorimeter : 1
+	    # includeCosmicRayVeto: 1
+	    includeCosmicRayVeto: 0
+	    includeDMAHeaders: 1
+	    # Cosmic ray veto module is currently incomplete
+
+	    generateTimestampTable : 1
+	    generateBinaryFile : 1
+	    
+	    outputFile      : "DTC_packets_calo.bin"
+	    maxDMABlockSize : 32000
+	    bufferSize      : 1000
+	}
```

Comment:
Consider moving the standard module definitions into a prolog sooner rather than later.